modules/appeals_api/app/models/appeals_api/supplemental_claim.rb

```diff
@@ -4,9 +4,11 @@
 
 module AppealsApi
   class SupplementalClaim < ApplicationRecord
```

Comment:
I kind of wish these models inherited from an `Appeal` model rather than `ApplicationRecord` so that `Appeal` could hold the data and behavior belonging to all appeal types, but I'm okay with the concern approach given that we haven't been following an inheritance pattern up to this point.